src/ds/ds5/ds5-color.cpp

```diff
@@ -217,7 +217,6 @@ namespace librealsense
         else
         {
             // Work-around for discrepancy between the RGB YUYV descriptor and the parser . Use UYUV parser instead
-            color_ep.register_processing_block(processing_block_factory::create_pbf_vector<uyvy_converter>(RS2_FORMAT_UYVY, map_supported_color_formats(RS2_FORMAT_UYVY), RS2_STREAM_COLOR));
```

Comment:
Is it for D457 only? 
 Pls you check that it was not D405-related as well.

Reply:
yes, please see if statement few lines above